SUPPORT.md

```diff
@@ -0,0 +1,94 @@
+
+Similar to a best-effort network, the capacity for our community as a whole to provide quality support for community members, and a welcoming environment for all contributors, depends heavily on the grace and good behavior of individual community members.
+
+You can help ensure a higher quality of best-effort support by formulating inquiries thoughtfully and making a considerate effort to place them in the most appropriate venue.
+
+Here are some guidelines about which venue makes the most sense:
+
+| Form of inquiry                                             | Venue                 |
+| ------------------------------------------------------------| --------------------- |
+| Something is not working as intended / I found a bug        | [Issue](https://github.com/fluxcd/flux2/issues) |
+| A feature is too limited for my `<use-case>`                | Discussion ([General](https://github.com/fluxcd/flux2/discussions/categories/general)) |
+| I want Flux to be able to do `<x>`                          | Discussion ([Proposal](https://github.com/fluxcd/flux2/discussions/categories/proposals)) |
+| Something is not working, (but I am not sure if I am doing it right) | Discussion ([Q&A](https://github.com/fluxcd/flux2/discussions/categories/q-a)) |
+| Quick question                                              | [#flux][] on [CNCF Slack][] |
+
+Bearing in mind that Issues and [Discussions](https://github.com/fluxcd/flux2/discussions) are more permanent and searchable than Slack conversations, we can avoid unduly expending finite community resources by searching before asking. If you are not exactly sure how to ask your question or otherwise daunted by the idea of permanence, visitors are always welcome in [#flux][] on the [CNCF Slack][].
+
+If your needs are more urgent, more broadly demanding, or more persistent than the best-effort community support resources can provide, you may also consider [a paid support option](#my-employer-needs-additional-help).
+
+## Community Help Resources
+
+For questions around Flux v2, please visit our [Flux2 Discussions](https://github.com/fluxcd/flux2/discussions) section on Github.
+
+## I found a bug
+
+If you made sure you encountered an actual issue, we definitely want to hear about it.
+
+Here's how to proceed:
+
+1. Check [Github](https://github.com/fluxcd)
+
+```txt
+TODO:
+- how to submit clear information if you really found an issue
+- figure out which project under `fluxcd` to file the issue on
+```
+
+## My employer needs additional help
+
+Luckily some of the companies who employ Flux developers offer paid support, so if you need an architecture review, training or help implementing certain features, you might want to reach out to the following companies:
+
+- Weaveworks: <https://www.weave.works/contact/>
```

Comment:
@dholbach could you add Xenit to this list?
- Xenit: <https://xenit.se/kontakt/>

Reply:
cc @fluxcd/oversight-committee

Reply:
:heavy_plus_sign: from me as an Oversight Committee member.

Reply:
Probably better to move this to a dedicated PR, makes voting more clear.

Reply:
Sure I will create a PR instead after this is merged to make the process simpler.